images/linux/scripts/installers/2004/android.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+################################################################################
+##  File:  android.sh
+##  Desc:  Installs Android SDK
+################################################################################
+
+set -e
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/document.sh
+source $HELPER_SCRIPTS/apt.sh
+
+# Set env variable for SDK Root (https://developer.android.com/studio/command-line/variables)
+ANDROID_ROOT=/usr/local/lib/android
+ANDROID_SDK_ROOT=${ANDROID_ROOT}/sdk
+echo "ANDROID_SDK_ROOT=${ANDROID_SDK_ROOT}" | tee -a /etc/environment
+
+# ANDROID_HOME is deprecated, but older versions of Gradle rely on it
+echo "ANDROID_HOME=${ANDROID_SDK_ROOT}" | tee -a /etc/environment
+
+# Create android sdk directory
+mkdir -p ${ANDROID_SDK_ROOT}
+
+# Download the latest command line tools so that we can accept all of the licenses.
+# See https://developer.android.com/studio/#command-tools
+wget -O android-sdk.zip https://dl.google.com/android/repository/sdk-tools-linux-4333796.zip
+unzip android-sdk.zip -d ${ANDROID_SDK_ROOT}
+rm -f android-sdk.zip
+
+# Add required permissions
+chmod -R a+X ${ANDROID_SDK_ROOT}
+
+# Check sdk manager installation
+/usr/local/lib/android/sdk/tools/bin/sdkmanager --list 1>/dev/null
+if [ $? -eq 0 ]
+then
+    echo "Android SDK manager was installed"
+else
+    echo "Android SDK manager was not installed"
+    exit 1
+fi
+
+# Install the following SDKs and build tools, passing in "y" to accept licenses.
+echo "y" | ${ANDROID_SDK_ROOT}/tools/bin/sdkmanager \
+    "ndk-bundle" \
+    "platform-tools" \
+    "platforms;android-29" \
+    "platforms;android-28" \
+    "platforms;android-27" \
+    "build-tools;29.0.3" \
```

Comment:
can we leave only one build-tools of each major version? or are all of them necessary?

Reply:
Customers ask for specific versions for some reason, like here when we have 29.0.3, but 29.0.1 needed
https://github.com/actions/virtual-environments/issues/344